.travis.yml

```diff
@@ -3,7 +3,11 @@ rvm:
 - 2.2.2
 gemfile:
 - spec/dummy/Gemfile
-install: bundle install
+install:
+- bundle install
+- cd spec/dummy && bundle install
```

Comment:
@mapreal19 I totally agree with @justin808 . We start with integration level is better because a lot of changes haven't merged, but the functionality should be the same.